apps/api/app/utils/cron_utils.py

```
@@ -15,6 +18,43 @@ class CronError(Exception):
+    offset_match = re.match(r"^([+-])(\d{2}):(\d{2})$", user_timezone)
+    if offset_match:
+        sign = 1 if offset_match.group(1) == "+" else -1
+        hours = int(offset_match.group(2))
+        minutes = int(offset_match.group(3))
+        offset_delta = timedelta(hours=hours, minutes=minutes)
+        return timezone(sign * offset_delta)
+
+    # Try IANA timezone name with pytz
+    try:
+        return pytz.timezone(user_timezone)
+    except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)